src/mon.c

```diff
@@ -2151,13 +2151,22 @@ register struct monst *mtmp;
         struct obj* list = collect_all_transient(NULL);
         struct obj* next;
         if (list) {
-            if (Blind && !Deaf)
-                You("hear a rush of objects jumble together nearby.");
+            /* first we tell the news to folks who can hear */
+            if (!Deaf) {
+                pline("A voice echoes in the arena:");
+                verbalize("Choose thy trophy from the spoils!");
+            }
+            /* then we noisily pile the objects together */
+            if (!Blind)
+                You("see %s possessions jumble together nearby.",
```

Comment:
Why is this changing the string from "coalesce into a large pile"?

Reply:
so that the messages for different senses are more similar.  it's weird that things "coalesce" when you can see them but "jumble" when you can't; those verbs have pretty different wordfeel!

Reply:
There are slightly different connotations at play between seeing things jumble and hearing things jumble. The visual message reads to me like their possessions are already in one place and just decided to move around a bit and shuffle positions, whereas "coalesce" has a distinct meaning of things coming together on one point.